src/contents/ui/main.qml

```diff
@@ -85,6 +85,8 @@ Item {
 	Plasmoid.compactRepresentation: HtmlClock { }
 	Plasmoid.fullRepresentation: CalendarView { }
 
+	Plasmoid.backgroundHints: plasmoid.configuration. transparentBackgroundEnabled ? "NoBackground" : "StandardBackground"
```

Comment:
There's a space between `configuration.` and `transparentBackgroundEnabled`. Not sure if that's valid syntax but is definitely bad practice.

Also, the background hints requires an Enum (a fancy integer) type, not a string.

https://api.kde.org/frameworks/plasma-framework/html/classPlasma_1_1Types.html#ab2b1c1767f3f432a0928dc7ca6b3f29e

As of Plasma 5.19(?), plasma added support for all desktop widget to have a transparent background in the resize handle. You just need to add the `ConfigurableBackground` hint with whatever default background hint you want.

* https://phabricator.kde.org/file/data/o2pejl4goxsr7j73kkgi/PHID-FILE-aw6ypyga4mkpl2a3d3rf/Screenshot_20191128_145617.png

Oddly, the KDE documentation has a bug. It mentions `ConfigurableBackground` at the top of the page but `ConfigurableBackground` is missing in the `BackgroundHints` table down below.

```
BackgroundHints {
  NoBackground = 0, StandardBackground = 1, TranslucentBackground = 2, ShadowBackground = 4,
  ConfigurableBackground = 8, DefaultBackground = StandardBackground
}
```

* https://api.kde.org/frameworks/plasma-framework/html/plasma_8h_source.html#l00280
* https://invent.kde.org/frameworks/plasma-framework/-/blame/master/src/plasma/plasma.h#L298

Anyways, for an example of a `ConfigurableBackground`, look at the default widgets.

* https://invent.kde.org/plasma/plasma-workspace/-/blob/master/applets/digital-clock/package/contents/ui/main.qml#L34

Reply:
Note that since it's only a year old, you may need to check if the enum value exists. Eg:

https://github.com/Zren/plasma-applet-githubissues/blob/master/package/contents/ui/Main.qml#L20

Reply:
Thanks for the feedback! Really appreciate. As for Plasma 5.19+ transparent background support - is this also available when plasmoid sits in Panel (as you do not have resize handles in such cases)?

Reply:
I don't think you can make the panel popup transparent... I do remember doing it once to TiledMenu though so I'm not sure.

https://www.google.com/search?q=site%3Agithub.com%2Fkde+backgroundHints

Plasma's dialogs have their own `PlasmaQuick::Dialog::BackgroundHint` enum which is different from the `Plasma::Types::BackgroundHints` that the `AppletInterface` (aka `Plasmoid.__`) uses.

* https://github.com/KDE/plasma-framework/blob/master/src/plasmaquick/dialog.h#L159
* https://github.com/KDE/plasma-framework/blob/master/src/scriptengines/qml/plasmoid/appletinterface.h


Unfortunately, widgets cannot access the "popup dialog". Well they can, but it's more of a hack than a defined API.

* https://github.com/Zren/plasma-applet-tiledmenu/issues/44